tests/foreman/cli/test_user.py

```diff
@@ -106,10 +105,7 @@ def test_bugzilla_1079649_2(self):
 
     @data(
         {'login': gen_string("latin1", 10)},
-        {
-            'login': gen_string("utf8", 10),
-            'bugzilla': 1144162,
-        },
+        {'login': gen_string("utf8", 10)},
```

Comment:
I'd argue that there's a couple good reasons to drop the `length` argument from each `gen_string` method:

* It's easy to discover the default values for each method. That's one major reason for the [API documentation](http://fauxfactory.readthedocs.org/en/latest/api.html).
* Omitting redundant arguments makes it easier to accept changes in default arguments. Let's say that upstream decides to give `gen_iplum` a default value for its `words` argument. (And that's probably a good change to make, actually.) Changing every instance of `gen_iplum(None, None)` to `gen_iplum(100, None)` will be a pain.
* Listing default arguments can get pretty ridiculous. For example: `reqeuests.request(method=None, url=None, headers=None, files=None, data=None, params=None, auth=None, cookies=None, hooks=None, json=None)`.